.ci/integrationTestEC.groovy

```diff
@@ -88,6 +88,11 @@ pipeline {
                   stash allowEmpty: true, includes: "${EC_DIR}/ansible/build/config_secrets.yml", name: "secrets-${getElasticStackVersion()}"
                 }
               }
+              stage("Waiting for services to settle") {
+                  steps {
+                      sleep 60
```

Comment:
As a follow-up, we could make this value configurable as a jenkins input parameter, so that we can play with that value until getting the most accurate one.

Reply:
The reason I didn't do that is that I want to be able to easily correlate between wait times and test results. Making it parametrized complicates that IMHO.